masterlist.yaml

```diff
@@ -2501,16 +2530,49 @@ plugins:
   - name: 'AOS.esp'
     after:
       - 'WetandCold.esp'
+      # RWT must be loaded after Requiem but AOS shouldn't so this rule is disabled when Requiem is installed
```

Comment:
RWT must be loaded after Requiem and AOS must be loaded after RWT unless the AOS-RWT patch is installed. These rules are in the masterlist. Logically if you use all three mods together AOS is loaded after Requiem but it shouldn't. In that case AOS should fall behind RWT and Requiem and the patch should be used.